sonic-net/sonic-buildimage
# [minigraph-parser] Update the definition of acl table type BMCDATA and BMCDATAV6

#### Why I did it

Update the definition of acl table type `BMCDATA` and `BMCDATAV6` in minigraph parser.

##### Work item tracking
- Microsoft ADO **(number only)**: 24101023

#### How I did it

Update the definition of acl table type `BMCDATA` and `BMCDATAV6` in minigraph parser.

#### How to verify it

Ran unittest to verify this update:

```
$ pytest-3 -v test_minigraph_case.py
================================================= test session starts ==================================================
platform linux -- Python 3.7.3, pytest-3.10.1, py-1.7.0, pluggy-0.8.0 -- /usr/bin/python3
cachedir: .pytest_cache
rootdir: /sonic/src/sonic-config-engine, inifile:
plugins: pyfakefs-5.2.2, cov-2.6.0
collected 42 items

test_minigraph_case.py::TestCfgGenCaseInsensitive::test_additional_json_data PASSED                              [  2%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_dhcp_table PASSED                                        [  4%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_dummy_run PASSED                                         [  7%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_jinja_expression PASSED                                  [  9%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_metadata_kube PASSED                                     [ 11%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_metadata_ntp PASSED                                      [ 14%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_metadata_tacacs PASSED                                   [ 16%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_acl_attach_to_ports PASSED                     [ 19%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_acl_type_bmcdata PASSED                        [ 21%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_bgp_mon PASSED                                 [ 23%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_cluster PASSED                                 [ 26%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_console_mgmt_feature PASSED                    [ 28%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_console_port PASSED                            [ 30%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_deployment_id PASSED                           [ 33%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_dhcp_server_feature PASSED                     [ 35%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_interfaces PASSED                              [ 38%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_mgmt_port PASSED                               [ 40%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_mirror_dscp PASSED                             [ 42%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_mux_cable_table PASSED                         [ 45%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_neighbor_metadata PASSED                       [ 47%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_peer_switch PASSED                             [ 50%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_peer_switch_hostname PASSED                    [ 52%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_portchannels PASSED                            [ 54%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_rack_mgmt_map PASSED                           [ 57%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_sku PASSED                                     [ 59%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_storage_backend_no_resource_type PASSED        [ 61%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_storage_backend_resource_type PASSED           [ 64%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_storage_backend_subintf PASSED                 [ 66%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_storage_device PASSED                          [ 69%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_subtype PASSED                                 [ 71%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_tunnel_table PASSED                            [ 73%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vlan_interfaces PASSED                         [ 76%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vlan_interfaces_keys PASSED                    [ 78%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vlan_members PASSED                            [ 80%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vlans PASSED                                   [ 83%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vnet PASSED                                    [ 85%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_minigraph_vxlan PASSED                                   [ 88%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_mux_cable_parsing PASSED                                 [ 90%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_parse_device_desc_xml_mgmt_interface PASSED              [ 92%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_print_data PASSED                                        [ 95%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_read_yaml PASSED                                         [ 97%]
test_minigraph_case.py::TestCfgGenCaseInsensitive::test_render_template PASSED                                   [100%]

============================================== 42 passed in 25.16 seconds ==============================================
```



#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
